src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -77,7 +77,7 @@
 
 /* Configuration Constants */
 #define SF1XX_BUS 		PX4_I2C_BUS_EXPANSION
-#define SF1XX_BASEADDR 	0x55
+#define SF1XX_BASEADDR 	0x66
```

Comment:
This means that the default address of SF10a is not used anymore. Additionally, I wonder why 0x55 did no give any spikes earlier then.